backend/src/main/java/com/example/vehiclesservice/api/Address.java

```diff
@@ -0,0 +1,18 @@
+package com.example.vehiclesservice.api;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data @NoArgsConstructor@AllArgsConstructor@Builder
```

Comment:
It's more readable when these are in multiple lines

Reply:
Also, are both constructors needed?